### ISSUE TITLE
Форма ввода проекта. Команда "Отмена"

### ISSUE THREAD

**Issue by shutilin** [vliplianina/scrum-team-2a#14]
Необходимо ли дополнительное сообщение (диалоговое окно) требующие подтверждения от пользователя после нажатия кнопки "Отмена"

**Comment by vliplianina** [comment]
Нет, запись удаляется без дополнительного уведомления пользователя.
При тестировании предлагаю оформить запросом на улучшение.

Замечание по оформлению: 
заголовок не отражает суть запроса. Звучит как задача на разработку. Прошу переформулировать.

**Comment by shutilin** [comment]
@vliplianina Переформулировал заголовок issue
